README.md

```diff
@@ -1,112 +1,137 @@
-# preCICE example for FSI: 1D elastic tube
+---
+title: 1D Elastic Tube
+permalink: unknown
+keywords: OpenFOAM, python
+summary: The 1D Elastic Tube is a FSI case, that consists of an internal flow in a flexible tube. The flow is unsteady and incompressible. This tutorial can be run using C++ or python.  Running the simulation takes a few minutes. 
+---
 
-## First Step: Choose a version
 
-This tutorial comes in two distinct versions, one written in C++ and one in Python. The code for each are located in the folders `cxx` and `python` respectively.
+## Setup
 
-Choose a version, then navigate down to the corresponding sections for [Python](#python-version) and [C++](#c-version) instructions.
+ALTERNATIVE TO THE LINK:
 
----
-## Python version
+We want to simulate the internal flow in a flexible tube as shown in the figure below (image from [1]).
 
-This version is realized using the Python API for preCICE. Check [this entry in the preCICE wiki](https://github.com/precice/precice/wiki/1D-elastic-tube-using-the-Python-API) for more information on this example.
+![FSI3 setup](images/setup.png)
 
-### Requirements
+Fluid is assumed to be incompressible flow and gravity is neglected. Due to the axisymmetry, the flow can be described using a quasi-two dimensional continuity and momentum equations. The motivation and exact formulation of the equations that we consider can be found in [2]. 
```

Comment:
```suggestion
The flow is assumed to be incompressible and gravity is neglected. Due to the axisymmetry, the flow can be described using a quasi two dimensional continuity and momentum equations. The motivation and exact formulation of the equations that we consider can be found in [2]. 
```
(I am very confused if it should be `quasi-two-dimensional`, `quasi two dimensional`, `quasi-two dimensional`, or `quasi two-dimensional`... :confused: )

Reply:
I found results with every combination but since I see that Nature uses `quasi-two-dimensional` I guess I can take this one as the good one.